src/functions-and-arrays.js

```diff
@@ -52,14 +133,41 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(uniqueArray) {
+
+  let simpleArray = [];
+  if(uniqueArray.length === 0){
+    return null;
+  } else {
+    uniqueArray.forEach((word) => {
+      if(simpleArray.includes(word) === false){
+       simpleArray.push(word);
+      }
+    });
+
+    return simpleArray;
+  }
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(findElement, target) {
+  if (findElement.length === 0){
+    return null;
+  } else if (findElement.length === 1){
```

Comment:
Here, you can use an array method called `includes()`, that is going to check if the word exist inside the array that we iterate over. The output of that method is `true` or `false`.
So, in this case, we can just:
`return findElement.includes(target)`